src/metabase/public_settings.clj

```diff
@@ -200,6 +200,18 @@
   :type    :boolean
   :default false)
 
+(defsetting enabled-persisted-models
```

Comment:
```suggestion
(defsetting enable-persisted-models
```

Seems weird to say "let's set `enabled-persisted-models` to `true`

Reply:
yeah but then conversely when reading the value it will sound weird. `(if (enable-persisted-models) ...)` reads as if it is attempting to enable rather than getting the value of whether persisted models are enabled or not. I think all of the settings will sound like that because they both read and write. prior art: `anon-tracking-enabled`, `ga-enabled`, etc. My regret is wondering why i didn't go with `persisted-models-enabled` and I don't know why i didnt.

Reply:
Yeah I would say "if enabled persisted models" doesn't exactly read naturally either. "Persisted models enabled" makes sense both as a vale and as something you would set tho